quesma/clickhouse/table_discovery.go

```diff
@@ -267,19 +275,20 @@ func (td *tableDiscovery) populateTableDefinitions(configuredTables map[string]d
 	for tableName, resTable := range configuredTables {
 		var columnsMap = make(map[string]*Column)
 		partiallyResolved := false
-		for col, colType := range resTable.columnTypes {
+		for col, columnMeta := range resTable.columnTypes {
 			if resTable.config.SchemaOverrides != nil {
 				if schemaOverride, found := resTable.config.SchemaOverrides.Fields[config.FieldName(col)]; found && schemaOverride.Ignored {
 					logger.Warn().Msgf("table %s, column %s is ignored", tableName, col)
 					continue
 				}
 			}
 			if col != AttributesValuesColumn && col != AttributesMetadataColumn {
-				column := resolveColumn(col, colType)
+				column := resolveColumn(col, columnMetadata{}.colType)
```

Comment:
```suggestion
				column := resolveColumn(col, columnMeta.colType)
```

Shouldn't it be this?